src/lib/pages/backoffice/Item/ItemDetails/ItemActionMenu/ItemActionMenu.js

```diff
@@ -113,16 +113,32 @@ export default class ItemActionMenu extends Component {
   };
 
   render() {
-    const { item, deleteItem, offset } = this.props;
+    const { item, deleteItem, offset, isInCirculation } = this.props;
     const hasCheckOutItem = this.hasCheckOutItem();
 
     return (
       <div className="bo-action-menu">
-        <EditButton
-          fluid
-          to={BackOfficeRoutes.itemEditFor(item.metadata.pid)}
-          text="Edit physical copy"
-        />
+        {isInCirculation ? (
+          <Popup
```

Comment:
Does it makes sense to create a `FakeButton` or `DisabledButton` component with this `PopUp` code and replace all the disables button with this, that way the user will have some feedback on why the button is disabled. WDYT ?

Reply:
Is this done because the Popup is not displayed on disabled buttons?
I think the button should be disabled.

Reply:
Yes, we can have the button disabled and make the Popup appear when we hover over. I thought it could be more user-friendly this way but I agree with you, thanks!